src/utils/sendUserDiscordData.ts

```diff
@@ -0,0 +1,37 @@
+import { env } from "../typeDefinitions/default.types";
+import jwt from "@tsndr/cloudflare-worker-jwt";
+import { BASE_URL } from "../constants/urls";
+
+export const sendUserDiscordData = async (
+  token: string,
+  discordId: number,
+  env: env
+) => {
+  const authToken = await jwt.sign(
+    { name: "Cloudflare Worker", exp: Math.floor(Date.now() / 1000) + 2 },
+    env.DISCORD_PRIVATE_KEY,
+    { algorithm: "RS256" }
+  );
+  const data = {
+    type: "discord",
+    token: token,
+    attributes: {
+      discordId: discordId,
+      expiry: Date.now() + 1000 * 60 * 2,
+    },
+  };
```

Comment:
as per the latest specification we also need to send the profile picture URL and user name. please add that

Reply:
Sure, thanks!